### PR TITLE
Add op paddle.device.cuda.get_device_name and paddle.device.cuda.get_device_capability.

### PR DESCRIPTION
**Pull request by liyagit21**
### PR types
New features 

### PR changes
OPs 

### Describe
Add op get_device_name and get_device_capability.
截屏2021-09-28 下午2 54 26
截屏2021-09-28 下午4 46 10
截屏2021-09-28 下午2 54 09
截屏2021-09-28 下午4 45 50

### CODE REVIEW

File: python/paddle/device/cuda/__init__.py [hunk 1]
```diff
@@ -271,3 +273,43 @@ def get_device_properties(device=None):
         device_id = -1
 
     return core.get_device_properties(device_id)
+
+
+def get_device_name(device=None):
+    '''
+    Return the name of the device which is got from CUDA function `cudaDeviceProp <https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__DEVICE.html#group__CUDART__DEVICE_1g1bf9d625a931d657e08db2b4391170f0>`_.
+    Parameters:
+        device(paddle.CUDAPlace|int, optional): The device or the ID of the device.
```
Comment (sneaxiy) [inline python/paddle/device/cuda/__init__.py]: Why is the format different with https://github.com/PaddlePaddle/Paddle/pull/35672/files#diff-f609e45ce88456728d295e8825b9b4b537dc8d73dc2034074efc59c4e937f170R283 ?

Same below.

**Review by sneaxiy — DISMISSED** [review]
LGTM.

**Review by TCChenlong — APPROVED** [review]

**Review by sneaxiy — APPROVED** [review]
LGTM.

**Review by jiangjiajun — APPROVED** [review]
LGTM

**Review by jiangjiajun — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

**Pull request merged**